### ISSUE TITLE
Update DT overlay to load gpio-ir

### ISSUE THREAD

**Issue by ChristopherRush** [PiSupply/Media-Center-HAT#9]

**Comment by shawaj** [comment]
Is this on the eeprom we need to update it?

Here it is for reference https://github.com/raspberrypi/linux/blob/rpi-4.19.y/arch/arm/boot/dts/overlays/media-center-overlay.dts

**Comment by tvoverbeek** [comment]
@shawaj @ChristopherRush Yes, the dts references lirc-rpi and lirc-pins.
lirc is outdated. Most of it ca now be done with the media framework which is part of the kernel.
The lirc-rpi module is no longer included in Buster.

**Comment by ChristopherRush** [comment]
I have added the overlay to the config.txt during the install process for the time being.

**Comment by shawaj** [comment]
Ahh ok - I guess we need to change the overlap in the main rpi repo then?

**Comment by tvoverbeek** [comment]
@shawaj Yes, More important which overlay is programmed in the Media HAT EEPROMs?

**Comment by shawaj** [comment]
@tvoverbeek the one in the eeprom currently is the GPIO 12 one. Using lirc.

Should have the final boards in about a week or two
